ykcompile/src/lib.rs

```diff
@@ -1417,6 +1634,7 @@ mod tests {
     }
 
     #[test]
+    #[ignore] // FIXME: Need to type our projections.
```

Comment:
Is just the test broken or is ref/deref in general broken now?

Reply:
When you have a deref on the left hand side of the assignment, we currently don't have a way to know its type. The same will be true for other projection types.

This can be fixed, but we don't need it right now, I don't think.

Reply:
Okay, we can fix this in a separate PR.